src/WebJobs.Script.WebHost/Security/ScriptSecretsType.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
```

Comment:
nit: remove unused usings. I see lots of these in other files as well, e.g. ScriptSecrets